### ISSUE TITLE
dock is not a function 

### ISSUE THREAD

**Issue by GregoryWullimann** [brightcove/videojs-dock#26]
I'm having problems using videojs-dock.

I'm working with VueJS and so far I didn't have any problem with other plugins, I'm using videojs-youtube and videojs-overlay and they work perfectly fine.

Now I'm trying to work with videojs-dock but I get that the dock function is not defined. I'm using the same code as the example:

```
import videojs from 'video.js';	
import 'videojs-dock';
import 'videojs-youtube';
import 'videojs-overlay';

player.dock({
        title: 'Bacon ipsum dolor amet ribeye',
        description: 'Doner shank pork loin sirloin, meatloaf shankle chicken bresaola cow shoulder cupim.'
});

```

And the error is
`VIDEOJS: ERROR: TypeError: this.player.dock is not a function`

VideoJS version: 7.1.0
Videojs-dock: 2.1.2
videojs-overlay: 2.1.3
videojs-youtube: 2.6.0

**Comment by misteroneill** [comment]
What tool are you using to bundle your web app - e.g. webpack, Rollup, etc?

**Comment by GregoryWullimann** [comment]
Webpack!

**Comment by GregoryWullimann** [comment]
Any updates? :)

**Comment by misteroneill** [comment]
@GregoryWullimann Oops! I forgot about this - should've written it down.

Anyway, I just published v2.1.3 from master. Can you try that version with your setup and let me know if it works for you?

**Comment by misteroneill** [comment]
Glad to hear it! Thanks for reporting the issue.
